src/collections/programs/Programs.style.js

```diff
@@ -1,132 +1,136 @@
 import styled from "styled-components";
 
 export const ProgramsWrapper = styled.div`
-    a {        color: ${props => props.theme.keppelColor};
-}
-    img{
-        width: 19vw;
-        float: right;
-        margin-left: 40px;
-        margin-bottom: 30px;
-        &.hacktoberfest-img {
-            width: 50%;
+  a {
```

Comment:
hey @saaagarsingh ig these changes are because of using some code formatter like Prettier which leads to removing the extra white space in the code. Please revert the unwanted changes.

Reply:
Hey, just wanted to know how can be revert back the changes once a PR has been raised. It would be helpful of you to suggest some solution.

Reply:
use git revert ( chatgpt can help u in that case🙂)

Reply:
...assuming that changes to be reverted were segregated into their own commit, yes.

Otherwise, this PR is quite small - just copy/paste and start over.